openerp/addons/base/ir/ir_translation.py

```diff
@@ -712,3 +712,23 @@ def load_module_terms(self, cr, modules, langs, context=None):
                     _logger.info('module %s: loading extra translation file (%s) for language %s', module_name, lang_code, lang)
                     tools.trans_load(cr, trans_extra_file, lang, verbose=False, module_name=module_name, context=context)
         return True
+
+
+    @api.model
+    def get_technical_translations(self, model):
```

Comment:
We are getting close :-)
Just a few more picky remarks:
 - one blank line between methods is enough
 - I would name the parameter `model_name` instead of `model`
 - please add a small docstring describing what the method returns